tests/js/spec/views/dashboardsV2/detail.spec.jsx

```diff
@@ -359,18 +365,31 @@ describe('Dashboards > Detail', function () {
       const card = wrapper.find('WidgetCard').first();
       card.find('StyledPanel').simulate('mouseOver');
 
-      // Edit the first widget
+      // // Edit the first widget
       wrapper
         .find('WidgetCard')
         .first()
         .find('IconClick[data-test-id="widget-edit"]')
         .simulate('click');
 
-      await tick();
-      await wrapper.update();
-      const modal = await mountGlobalModal();
-
-      expect(modal.find('AddDashboardWidgetModal').props().widget).toEqual(widgets[0]);
+      expect(openEditModal).toHaveBeenCalledTimes(1);
+      expect(openEditModal).toHaveBeenCalledWith(
+        expect.objectContaining({
+          widget: {
+            id: '1',
+            interval: '1d',
+            queries: [
+              {
+                conditions: 'event.type:error',
+                fields: ['count()'],
+                name: '',
+              },
+            ],
+            title: 'Errors',
+            type: 'line',
+          },
+        })
+      );
```

Comment:
Hmm, this change significantly reduced the run time of this test locally (~800ms -> 350ms) but made no difference on CI.

Local
```  Dashboards > Detail
    prebuilt dashboards
      ✓ can delete (325 ms)
      ✓ can rename and save (133 ms)
      ✓ disables buttons based on features (41 ms)
    custom dashboards
      ✓ can remove widgets (268 ms)
      ✓ can enter edit mode for widgets (378 ms)  <---
      ✓ does not update if api update fails (406 ms)
      ✓ shows add wiget option (187 ms)
      ✓ hides add widget option (162 ms)
      ✓ hides and shows breadcrumbs based on feature (206 ms)
      ✓ enters edit mode when given a new widget in location query (102 ms)
      ✓ enters view mode when not given a new widget in location query (94 ms)
      ✓ can add library widgets (289 ms)
      ✓ disables add library widgets when max widgets reached (76 ms)
      ✓ adds an Issue widget to the dashboard (560 ms)
```
CI
```
PASS tests/js/spec/views/dashboardsV2/detail.spec.jsx (17.832 s)
  Dashboards > Detail
    prebuilt dashboards
      ✓ can delete (909 ms)
      ✓ can rename and save (289 ms)
      ✓ disables buttons based on features (71 ms)
    custom dashboards
      ✓ can remove widgets (474 ms)
      ✓ can enter edit mode for widgets (5571 ms)  <--- still over 5000 ms!
      ✓ does not update if api update fails (752 ms)
      ✓ shows add wiget option (380 ms)
      ✓ hides add widget option (353 ms)
      ✓ hides and shows breadcrumbs based on feature (429 ms)
      ✓ enters edit mode when given a new widget in location query (214 ms)
      ✓ enters view mode when not given a new widget in location query (195 ms)
      ✓ can add library widgets (718 ms)
      ✓ disables add library widgets when max widgets reached (159 ms)
      ✓ adds an Issue widget to the dashboard (2079 ms)
```